### PR TITLE
Module 1 - Day 2 - Hello Microverse Page

### PR DESCRIPTION
**Pull request by lmunozm1702**
In this request I:

- Create the index.html page.
- Create the style.css file to set the color of the content in index.html.
- Set up .gitignore file to avoid tracking the test.md file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+test.md
```
Comment (aimalamiri) [inline .gitignore:2]: - [x] Good job! According to the project requirements, it's important to add the `node_moduels` directory to your `.gitignore` file, so make sure to include it.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+- HTML
+- CSS
+
+##Getting Started
+This project's about how to create a project on GitHub and set up a linter for HTML&CSS, and make a new branch to implement the "hello microverse" functionality.
+
+##Authors
+###Luis Muñoz
+
+- GitHub: @lmunozm1702
```
Comment (aimalamiri) [inline README.md]: - [x] Please add links to your GitHub profile so that people can visit your profile by clicking on the link.

**Review by aimalamiri — CHANGES_REQUESTED** [review]
Hi @lmunozm1702, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

AlmostThereReachGIF

## Highlights


## Required Changes ♻️

- [x] **Please make sure to add the `.hintrc` and the `.stylelintrc.json` files to your repository.**


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (lmunozm1702) [reply .gitignore:2]: It's done.

File: README.md (hunk 2 above)
Comment (lmunozm1702) [reply README.md]: It's done.

**Comment by lmunozm1702** [comment]
All code review issues (3) are solved.

**Comment by enniopo** [comment]
Hi @lmunozm1702, please request another code review request from the Dashboard. You should be able to do it now. Thanks 🙏

**Comment by lmunozm1702** [comment]
Hi @enniopo, I did the new request from the dashboard, thanks a lot!

**Review by mckent05 — APPROVED** [review]
Hi @lmunozm1702, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and good job! 🎉 

## Optional suggestions
**[OPTIONAL]** - Please note that in your future projects, having a descriptive PR title will help to make your repo look more professional. in this case, for example, your title could be something as short as "Hello Microverse Project" 😄 😄 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by lmunozm1702** [comment]
Hi! @mckent05, it's merged!!! Thanks a lot!!!

**Pull request merged**